v2/cmd/asoctl/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/Azure/azure-service-operator/v2/tools/asoctl
```

Comment:
Do we need to be consistent on where we have the go.mod for the `controller` as well?

Reply:
The `go.mod` and `go.sum` files have to be at the root of the project, so we don't have any option on where to put them.